charts/controller/templates/controller-ingress-rule-builder-2222.yaml

```diff
@@ -0,0 +1,21 @@
+{{ if .Values.global.experimental_native_ingress }}
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  namespace: "deis"
+  name: "controller-api-server-builder"
+  labels:
+    app: "controller"
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+spec:
+  rules:
+  - host: deis.{{ .Values.platform_domain }}
+    http:
+      paths:
+      - path: /
+        backend:
+          serviceName: deis-controller
+          servicePort: 2222
+{{- end }}
```

Comment:
Does this ingress rule end up having the intended effect? afaik, the builder speaks ssh, meaning the traffic flowing to it is raw TCP... whereas ingress _only_ works for HTTP/S. 😢 (How I wish that weren't so.)

I _think_ that when experimental native ingress is enabled, we have to make builder's service one of `type: LoadBalancer` to compensate for the fact that deis-router won't be around handling the non-HTTP builder traffic as a special case.